src/App.tsx

```diff
@@ -1,61 +1,135 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [currentTodos, setCurrentTodos] = useState<Todo[]>(todos);
+
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+
+  const [option, setOption] = useState(0);
```

Comment:
Two things here:

- `option` doesn't say much, something like `selectedUser` would be way better.

- For the default value (no user selected) I would use null
```suggestion
  const [selectedUser, setSelectedUser] = useState<number | null>(null);
```

Reply:
It kinda prevents code from starting on 0 field, that's 'chose a user', but it got implemented ;)

Reply:
Cant do that. Tried, but it makes it impossible to use proper starting state. Also Value cannot use null type